deno_dist/middleware/jwt/index.ts

```diff
@@ -13,6 +13,33 @@ declare module '../../context.ts' {
   }
 }
 
+/**
+ * JWT Auth middleware for Hono.
+ *
+ * @see {@link https://hono.dev/middleware/builtin/jwt}
+ *
+ * @param {object} options - The options for the JWT middleware.
+ * @param {string} options.secret - A value of your secret key.
```

Comment:
Is the following better?

```ts
@param {string} [options.secret] - A value of your secret key.
```

Reply:
@goisaki 

I added by myself: https://github.com/honojs/hono/pull/2709/commits/9d1cdf73a4dd38b5bb6245edf42aaef925bb33cd

Reply:
Thanks!